src/lib/cyleaflet_clientside.js

```diff
@@ -0,0 +1,84 @@
+if (!window.dash_clientside) {
+    window.dash_clientside = {};
+}
+
+// Functions to convert lat/lon used by Leaflet to x/y used by Cytoscape.
+//  - EPSG:4326 is the 'standard' (Mercator) grid coordinate system,
+//    which maps nicely to Cytoscape's x/y grid
+//  - EPSG:3857 is the 'Web Mercator' projection used by many
+//    online mapping serveces including Leaflet
+// Reference: https://epsg.io/3857
+
+// Conversion factor based on EPSG:3857 bounds
+var conversion_factor = 20037508.34;
+
+// Convert EPSG:4326 to EPSG:3857
+// We also flip the sign of the y-value to match Cytoscape's coordinate system
+function lonLatToXY(lon, lat) {
+    var x = lon * conversion_factor / 180;
+    var y = Math.log(Math.tan((90 + lat) * Math.PI / 360)) / (Math.PI / 180);
+    y = y * conversion_factor / 180;
+    return [x, -y];
```

Comment:
```suggestion
    var y = -Math.log(Math.tan((90 + lat) * Math.PI / 360)) * conversion_factor / Math.PI;
    return [x, y];
```
Just simplifying a little - and making it clearer that these two functions are inverses 😎